### PR TITLE
lift-wls-deployment-tool

### PR DESCRIPTION
**Pull request by Soumyadeep-Oracle**

### CODE REVIEW

File: cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/valiJSON.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  mv input.json.tmp input.json
+fi
+checkFalse=`grep "False" input.json`
+if [ $? -eq 0 ]; then
+  sed -e 's/"False"/"false"/' input.json > input.json.tmp
+  mv input.json.tmp input.json
+fi
+echo "Resultant JSON as follows"
+cat input.json
+echo -e "\n\nPlease validate JSON here if you want https://jsonlint.com/"
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/valiJSON.sh:21]: Remove this link. It is not EU compliant.

File: cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/wl_mpi_template_v2.xlsx
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/wl_mpi_template_v2.xlsx:1]: Clear cells
G2, G4, G6

**Review by oheimburger — CHANGES_REQUESTED** [review]

File: cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/wl_mpi_template_v2.xlsx
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/wl_mpi_template_v2.xlsx:1]: Submitted required changes.

File: cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/valiJSON.sh (hunk 1 above)
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/weblogic-deployment-utility/files/valiJSON.sh]: Submitted required changes.

**Review by oheimburger — APPROVED** [review]

**Pull request merged**